frontend/src/components/Election/Voting/BallotSelector.js

```diff
@@ -27,6 +29,22 @@ export default function BallotSelector({
         onUpdate={onUpdate}
         />
     }
+    {(race.voting_method == 'Ranked-Robin' || race.voting_method == "IRV") && 
```

Comment:
We'll probably need separate ballot view for RCV and Ranked Robin as the instructions and tabulation description will be different.

Reply:
I'm happy to do this in the future, but for the Ranked ballot view will update it's description depending on which method is being used

Reply:
Ah, didn't see that.